improver/wxcode/weather_symbols.py

```diff
@@ -430,6 +430,49 @@ def find_all_routes(
                 routes.extend(newroutes)
         return routes
 
+    @staticmethod
+    def check_for_time_bounds(cubes: Union[List[Cube], CubeList]) -> Cube:
```

Comment:
I've rejigged things. I've chosen to set the template cube itself as the global variable because the information is right there, so adding more code seems a bit needless, but feel free to disagree.

I've still got to look at the acceptance test data.